model/Core/Vocabularies/ProfileIdentifierType.md

```diff
@@ -4,28 +4,38 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-Enumeration of the valid profiles. 
+Enumeration of the valid profiles.
 
 ## Description
 
 There are a set of profiles that have been defined by a profile team.
-A profile consists of a namespace that may add properties and classes to the core profile unique to the domain covered by the profile.
-The profile may also contain additional restrictions on existing properties and classes defined in other profiles.
-If the creator of an SPDX collection of elements includes a profile in the list of conformanceProfiles, they are claiming that all contained elements conform to all restrictions defined for that profile.
+
+A profile consists of a namespace that may add properties and classes to the
+core profile unique to the domain covered by the profile.
+
+The profile may also contain additional restrictions on existing properties and
+classes defined in other profiles.
+
+If the creator of an SPDX collection of elements includes a profile in the list
+of conformanceProfiles, they are claiming that all contained elements conform
+to all restrictions defined for that profile.
 
 ## Metadata
 
 - name: ProfileIdentifierType
 
 ## Entries
 
-- core: the element follows the Core profile specification
-- software: the element follows the Software profile specification
-- simpleLicensing: the element follows the simple Licensing profile specification
-- expandedLicensing: the element follows the expanded Licensing profile specification
-- security: the element follows the Security profile specification
-- build: the element follows the Build profile specification
 - ai: the element follows the AI profile specification
+- build: the element follows the Build profile specification
+- core: the element follows the Core profile specification
 - dataset: the element follows the Dataset profile specification
-- usage: the element follows the Usage profile specification
+- expandedLicensing: the element follows the expanded Licensing profile
+  specification
 - extension: the element follows the Extension profile specification
+- licensing: the element follows the Licensing profile specification
```

Comment:
No, `licensing` is not a profile that should be used in a list of conformant profiles. Only `simpleLicensing` and `expandedLicensing` should be allowed.

Reply:
Thank you, it is now removed.